themes/default/content/docs/intro/cloud-providers/azure/version-guide.md

```diff
@@ -0,0 +1,143 @@
+keyvault.NewVault(/*...*/)
+// ...
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+As soon as Microsoft publishes a new API version, the corresponding module is created in the native Azure provider for Pulumi and will be published in the next minor release.
+
+Older API versions remain in the provider indefinitely. Therefore, you can pin all your resources to specific API versions. You can keep using them for as long as those versions are supported by Azure, even across major versions of the Pulumi Azure-Native provider.
+
+## Top-Level Modules
+
+In addition to all versioned modules described above, the Pulumi Azure-Native provider publishes a default version of every resource in top-level modules. For example, the `Vault` resource of the `KeyVault` resource provider is published as the Pulumi resource `azure-native:keyvault:Vault`. You can import this resource with the following statement:
+
+{{< chooser language "typescript,python,csharp,go" >}}
+
+{{% choosable language typescript %}}
+
+```typescript
+import Vault from "@pulumi/azure-native/keyvault";
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+from pulumi_azure_native.keyvault import Vault
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using Pulumi.AzureNative.KeyVault;
+
+new Vault(/*...*/);
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+import (
+    "github.com/pulumi/pulumi-azure-native/sdk/go/azure/keyvault"
+)
+
+// ...
+keyvault.NewVault(/*...*/)
+// ...
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Top-level resources are a convenience mechanism to simplify the experience of users that don't have a preference over which API version to use for every resource. We expect most users to use top-level resources most of the time, without the need to specify an explicit API version.
+
+Each top-level resource maps internally to a specific API version. The native Azure provider maintains this mapping with the following guidelines:
+
+- When a new major version of the provider is released, the API versions are set to the latest stable API version available at that moment. If no stable version is published yet, the newest preview version is used.
+- The selected API version is pinned across all minor versions of the same major version. Even if a newer API version is released, we do not promote the top-level resources to it.
+- We may introduce some manual overrides to the rules above. For example, we may use a new preview version when the latest stable version is obviously out of date. We may manually promote top-level resources to a new API version to get access to new necessary functionality. In any case, we adhere to the breaking changes policy.
+
+Because of that, top-level resources may miss newly released features shipped in brand-new API versions after the last major release. In such cases, users are encouraged to use explicitly versioned modules to access the latest Azure functionality.
```

Comment:
```suggestion
Because of that, top-level resources may miss newly released features shipped in brand-new API versions after the last major release. In such cases, you are encouraged to use explicitly versioned modules to access the latest Azure functionality.
```